weblate/static/editor/full.js

```diff
@@ -199,16 +199,11 @@
   FullEditor.prototype.initTabs = function () {
     /* Store active tab in a cookie */
     $('.translation-tabs a[data-toggle="tab"]').on("shown.bs.tab", function () {
-      const current = Cookies.get("translate-tab");
+      const current = window.localStorage.getItem("translate-tab");
```

Comment:
No need for calling `window.localStorage` from the window object. As `localStorage` is available on the global scoop by default.

Also in the `loader-bootstrap.js`.